externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -65,8 +65,10 @@ case class SparkSQLEngine(spark: SparkSession) extends Serverable("SparkSQLEngin
   }
 
   override def stop(): Unit = {
-    eventLogging.onEvent(
-      engineStatus.copy(state = ServiceState.STOPPED.id, endTime = System.currentTimeMillis()))
+    if (!state.equals(ServiceState.LATENT)) {
```

Comment:
should it be `if (state == ServiceState.STARTED)` ?

Reply:
i think state judgement should in the method of JsonEventLogger.getOrUpdate.
only "STARTED" state can provided this service.

this need @yaooqinn to confirm.